android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -233,11 +236,14 @@ constructor(
             questionnaireConfig = questionnaireConfig,
           )
 
-          executeCql(
-            subject = subject,
-            bundle = newBundle,
-            questionnaire = questionnaire,
-          )
+          withContext(Dispatchers.IO) {
```

Comment:
Can you use `dispatcherProvider.io()` here instead? The provider is already injected as a constructor parameter